node-update.sh

```diff
@@ -28,12 +29,18 @@ if [[ $SYSTDDVIRT = "wsl" ]];
         exit 1;
 fi;
 
+if [ -z $(type -P apt-get) ]
+        then
+        echo "Only a Debian-based Linux is supported"
+        unset LC_ALL;
+        exit 1;
+fi;
 
 ### Starting the skript
 echo -e "ioBroker node-update v$VERSION is starting. Please be patient!";
 HOST=$(hostname)
-NODERECOM=$(iobroker state getValue system.host."$HOST".versions.nodeNewestNext);  #reading node version from iob states. If successful, no fallback required.
-if [[ $NODERECOM != [[:digit:]]*.[[:digit:]]*.[[:digit:]]* ]]; #check if a semver'd nodejs installation is found
+NODERECOM=$(iobroker state getValue system.host."$HOST".versions.nodeNewestNext);  #reading node version from iob states. If succesfull, no fallback required.
```

Comment:
The previous version of `successful` was actually the correct one afaik

Reply:
Corrected.